### ISSUE TITLE
Testing Library Explorer Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4640]
# Page to Test:

Library Explorer - https://openlibrary.org/explore

# Steps

Steps for this task are outlined in #4521
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by melissa-tan10** [comment]
The alt attribute is labeled inconsistently.

Here it is labeled as "Internet Archive logo":
![image](https://user-images.githubusercontent.com/69818577/110158720-b7bc7300-7db7-11eb-9c21-1937d3d451bb.png)

In the rest of the file it is null:
![image](https://user-images.githubusercontent.com/69818577/110158634-9eb3c200-7db7-11eb-8ca6-e69b037fc402.png)

**Comment by Yashs911** [comment]
@melissa-tan10 According to https://github.com/internetarchive/openlibrary/issues/4521#issuecomment-772730099 the empty alt attr for the logo is fine.
@bpmcneilly Alt attr is missing from the book covers present on this page.
